zppy/templates/coupled_global.py

```diff
@@ -514,26 +709,54 @@ def run(parameters, rgn):  # noqa: C901
     year2 = int(parameters[5])
     color = parameters[6]
     ts_num_years = parameters[7]
-    if parameters[8].lower() == "false":
+    plots_original = param_get_list(parameters[8])
+    if parameters[9].lower() == "false":
         atmosphere_only = False
     else:
         atmosphere_only = True
-    plot_list = parameters[9].split(",")
+    plots_atm = param_get_list(parameters[10])
+    plots_lnd = param_get_list(parameters[11])
+    plots_ocn = param_get_list(parameters[12])
+    plots_sea_ice = param_get_list(parameters[13])
+    vars_original = [
+        "RESTOM",
+        "RESSURF",
+        "TREFHT",
+        "FSNTOA",
+        "FLUT",
+        "PRECC",
+        "PRECL",
+        "QFLX",
+    ]
+    use_atmos = plots_atm or plots_original
+    use_ocn = plots_ocn or (plots_original and not atmosphere_only)
     exps = [
         {
-            "atmos": "{}/post/atm/glb/ts/monthly/{}yr/glb.xml".format(
+            "atmos": None
+            if not use_atmos
+            else "{}/post/atm/glb/ts/monthly/{}yr/glb.xml".format(
+                case_dir, ts_num_years
+            ),
+            "land": None
+            if not plots_lnd
+            else "{}/post/lnd/glb/ts/monthly/{}yr/glb.xml".format(
                 case_dir, ts_num_years
             ),
             "ocean": None
-            if atmosphere_only
+            if not use_ocn
             else "{}/post/ocn/glb/ts/monthly/{}yr/glb.xml".format(
                 case_dir, ts_num_years
             ),
+            "sea_ice": None
+            if not plots_sea_ice
+            else "{}/post/sea_ice/glb/ts/monthly/{}yr/glb.xml".format(
+                case_dir, ts_num_years
+            ),  # TODO: check sea_ice path is correct
```

Comment:
Question 4) Sorry, I missed this in my initial self-review. Is the path for sea-ice just `sea_ice`? I don't think I've seen a simulation that includes it. I see `ice` on both simulations I tested on -- is that sea ice or land ice??

Reply:
Fixed.